File: Python/Kode/petshop.py
class petshop:
    def __init__(self, id="", nama="", brand="", harga=0, stok=0):
        self.__id = id
        self.__nama_produk = nama
        self.__brand = brand
        self.__harga_produk = harga
        self.__stok_produk = stok
    
    def get_id(self):
        return self.__id
    
    def set_id(self, id):
        self.__id = id
    
    def get_nama_produk(self):
        return self.__nama_produk
    
    def set_nama_produk(self, nama):
        self.__nama_produk = nama
    
    def get_brand(self):
        return self.__brand
    
    def set_brand(self, brand):
        self.__brand = brand
    
    def get_harga_produk(self):
        return self.__harga_produk
    
    def set_harga_produk(self, harga):
        self.__harga_produk = harga
    
    def get_stok_produk(self):
        return self.__stok_produk
    
    def set_stok_produk(self, stok):
        self.__stok_produk = stok
    
    def tampilkan_data(self):
        print(f"| {self.__id:<6} | {self.__nama_produk:<18} | {self.__brand:<13} | {self.__harga_produk:<8.0f} | {self.__stok_produk:<4} | ", end="")

File: Java/Kode/Petshop.java
public class Petshop {
    private String id;
    private String namaProduk;
    private String brand;
    private double hargaProduk;
    private int stokProduk;
    
    // Constructor
    public Petshop(String id, String namaProduk, String brand, double hargaProduk, int stokProduk) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.brand = brand;
        this.hargaProduk = hargaProduk;
        this.stokProduk = stokProduk;
    }
    
    // Default constructor
    public Petshop() {
        this("", "", "", 0, 0);
    }
    
    // Getter and setter for id
    public String getId() {
        return id;
    }
    
    public void setId(String id) {
        this.id = id;
    }
    
    // Getter and setter for namaProduk
    public String getNamaProduk() {
        return namaProduk;
    }
    
    public void setNamaProduk(String namaProduk) {
        this.namaProduk = namaProduk;
    }
    
    // Getter and setter for brand
    public String getBrand() {
        return brand;
    }
    
    public void setBrand(String brand) {
        this.brand = brand;
    }
    
    // Getter and setter for hargaProduk
    public double getHargaProduk() {
        return hargaProduk;
    }
    
    public void setHargaProduk(double hargaProduk) {
        this.hargaProduk = hargaProduk;
    }
    
    // Getter and setter for stokProduk
    public int getStokProduk() {
        return stokProduk;
    }
    
    public void setStokProduk(int stokProduk) {
        this.stokProduk = stokProduk;
    }
    
    // Method to display data
    public void tampilkanData() {
        System.out.printf("| %-6s | %-18s | %-13s | %-8.0f | %-4d | ", id, namaProduk, brand, hargaProduk, stokProduk);
    }
}